### ISSUE TITLE
Fail to install: Python 3.9

### ISSUE THREAD

**Issue by majiang** [taoufik07/responder#442]
The following commands,
```
docker run -it python:3.9 bash
pip install responder
```
which worked with python:3.8, gave errors when installing `httptools`.  The fix is [already in `uvicorn`](https://github.com/encode/uvicorn/commit/6598b0540b00a39b51c972b8ea9f8083168b16c3).  I suspect that this problem will be fixed once you add a tag for master branch (for the first time in 9 months).

**Comment by fenchu** [comment]
Tried responder 2.0.5 from Dec15, but:
```
Collecting httptools==0.0.13
  Using cached httptools-0.0.13.tar.gz (104 kB)
```
You need httptools==0.1.1 on python3.9

**Comment by waghanza** [comment]
@taoufik07 seems to be done on **master**, and fix this issue. any release planned ?

**Comment by majiang** [comment]
Is there anything the community can help releasing?

**Comment by taoufik07** [comment]
A new release with python 3.9 support has been published 2.0.6  🎉

**Comment by majiang** [comment]
Thanks! I'll try it out.
